Section8/src/MerArrays.java

```diff
@@ -0,0 +1,51 @@
+import java.util.Scanner;
+import java.util.Arrays;
+
+
+public class MerArrays {
+
+    public static void main(String[] args) {
+        int[] myIntArray = new int[10];
+        myIntArray[0] = 45;
+        int[] myArray = {1, 2, 3, 4, 5, 6, 7, 8, 9, 10};
+        System.out.println(Arrays.toString(myArray)); // ???????
```

Comment:
(Non critical) always consider removing comments that don't convey any meaningful information before a commit.